### ISSUE TITLE
Unity mono debug assemblies need updating

### ISSUE THREAD

**Issue by Kein** [0xd4d/dnSpy#1263]
Got game for 2018.4.7 but supplied debug mono assemblies go only as high as `unity-2018.4.3`

**Comment by Kein** [comment]
While I'm at it - the umpatcher seems to be broken:
```
Run umpatcher again to patch the code and commit it to the dnSpy-Unity-mono repo
umpatcher 5.4.3 aa8a6e7afc2f4fe63921df4fe8a18cfd0a441d19 "C:\path\to\Unity-mono" "C:\path\to\dnSpy-Unity-mono"
```
Results in:
```
umpatcher.exe 2018.4.7 cef4a8d2357178ca3c5a613341a53f0f3b67113a "g:\Temp\Unity\mono\" "g:\Temp\Unity\dnSpy-Unity-mono\"

Invalid command line arguments
```

**Comment by Kein** [comment]
Well, I give up. I tried to make project manually by copying necessary files but I cant add and/or modify libmono because it fails to import it, complaining about filters. I cant even build existing mono libs due to the fact I cant retarged same project to 2019, so, even if `umpatcher` did work properly it still wouldnd compile.

Man, this shit is so needlessly and unnecessary overcomplex, really reminds me why C++ project architecture and management makes me puke. **If you can add latest two 2018 (4.7, 4.8 and 4.9) debug mono versions to the archive that'd be appreciated**

**Comment by 0xd4d** [comment]
Have you tried the latest built 2018.4.x binaries? They're usually very similar so could work.

**Comment by Kein** [comment]
To be fair dont want to second-guess what could be an issue with debugging yet again, dnspy debugging is imprecise and unstable as it is. Better do not touch the castle of cards.

**Comment by 0xd4d** [comment]
Well then it's nothing I can do.

**Comment by Kein** [comment]
You mean you **dont want to do**, right? Since I'm confident you easily **can** just add 3 more versions of patched mono for 2018.6,7,8. as you did for the rest of the bunch.

**Comment by 0xd4d** [comment]
Welcome to banville.
